fetch-jars

```diff
@@ -3,26 +3,62 @@
 set -eu
 
 # Usage: ./fetch-jars [rev-name] [--copper]
+# If a revision is not specified, the script fetches the latest jars from the current branch,
+# falling back to develop.
+# If --copper is specified, only fetch the Copper jars.
 
+COMMIT_ARTIFACTS="https://foundry.remexre.xyz/commit-artifacts/"
 
+function has_jars {
+  wget --spider -q "$COMMIT_ARTIFACTS/$1"
+}
+
+rev=""
+
+# Look for jars in the specified revision
 if [[ $* != *--copper* && $# -gt 0 || $# -gt 1 ]]; then
```

Comment:
Is this check right? seems like it'd ignore the rev if you did `./fetch-jars v0.4.4 --copper`

Reply:
...except that it's missing parens too, I think?

Reply:
`./fetch-jars v0.4.4 --copper` should work? Since the number of args is greater than 1. I didn't update this logic, though.